exporters/jaeger/src/main/java/io/opentelemetry/exporters/jaeger/Adapter.java

```diff
@@ -192,7 +196,10 @@ private Adapter() {}
   @VisibleForTesting
   static <T> Model.KeyValue toKeyValue(AttributeKey<T> key, T value) {
     Model.KeyValue.Builder builder = Model.KeyValue.newBuilder();
-    builder.setKey(key.getKey());
+    builder.setKey(
+        key.getKey()
+            .replace("InstrumentationLibrary.name", "otel.library.name")
+            .replace("InstrumentationLibrary.version", "otel.library.version"));
```

Comment:
Why is this needed?

Reply:
Thanks @malafeev - looks like there was some confusion on the issue, yeah you shouldn't need to do any sort of name translation logic.

Reply:
removed

Reply:
I was confused by https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/sdk_exporters/jaeger.md#resource
I decided that `InstrumentationLibrary.name` should be translated into `otel.library.name` etc